foundation/foundation-upgrade/src/test/resources/test-files/v1_8_0/AiopsReferenceJavaMigration/validation/IngestValidated.java

```diff
@@ -63,13 +63,13 @@
  *
  * Generated from: templates/data-delivery-spark/synchronous.processor.base.java.vm
  */
-public abstract class Ingest extends AbstractPipelineStep {
+public abstract class IngestValidated extends AbstractPipelineStep {
```

Comment:
Is this file used/necessary? Don't see it referenced in the test steps anywhere.

Reply:
This was a test file from #194. There was a recommendation to rename something, however after testing the change it was reverted and removed from this PR.